packages/payments-plugin/src/mollie/custom-fields.ts

```diff
@@ -0,0 +1,16 @@
+import { CustomFieldConfig, Order, CustomOrderFields } from '@vendure/core';
+
+export interface OrderWithMollieReference extends Order {
+    customFields: CustomOrderFields & {
+        mollieOrderId?: string;
+    };
+}
```

Comment:
Use a custom interface instead of a `*-types.d.ts` file, because that would also infer `mollieOrderId` in the Stripe, Paypal and braintree plugin.

Reply:
OK, makes sense to encapsulate this implementation detail 👍